web/src/poll_modal.ts

```diff
@@ -46,8 +46,10 @@ export function poll_options_setup() {
 
     // setTimeout is needed to here to give time for simplebar to initialise
     setTimeout(() => {
-        Sortable.create($("#add-poll-form .poll-options-list .simplebar-content")[0], {
-            onUpdate() {},
+        new SortableJS($("#add-poll-form .poll-options-list .simplebar-content")[0], {
+            onUpdate() {
+                // do nothing
```

Comment:
[eslint-no-empty-function](https://eslint.org/docs/latest/rules/no-empty-function)

Reply:
The intent of that eslint rule is for you to put a comment explaining **why** doing nothing is correct. The "do nothing" comment is just as useless as not having one :)

Reply:
Added a comment to resolve this,.

Reply:
Thanks ! . Noted for future work.